mu-plugins/blocks/global-header-footer/admin-bar.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace WordPressdotorg\MU_Plugins\Global_Header_Footer;
+
+use Rosetta_Sites, WP_Post, WP_REST_Server, WP_Theme_JSON_Resolver;
+
+defined( 'WPINC' ) || die();
+
+/* Actions & filters */
+add_action( 'admin_bar_menu', __NAMESPACE__ . '\filter_admin_bar_links', 500 ); // 500 to run after all items are added to the menu.
+add_filter( 'show_admin_bar', __NAMESPACE__ . '\should_show_admin_bar' );
+
+/**
+ * Always show the admin bar.
+ *
+ * @param bool $show_admin_bar Whether the admin bar should be shown.
+ * @return bool
+ */
+function should_show_admin_bar( $show_admin_bar ) {
+	// @todo Don't enable on the login page.
```

Comment:
I'm not sure how to target the login site to exclude it, but also it seems like [this code should be disabling it](https://github.com/WordPress/wordpress.org/blob/c54eb93f3f27d02bba495f6a418db2316e9a03c6/wordpress.org/public_html/wp-content/themes/pub/wporg-login/functions.php#L47). Suggestions welcome 🙂

Reply:
This is because the filter used here runs on the default, and `show_admin_bar()` as used in the login theme only toggles the default state, the filter can still override it.

I wasn't able to duplicate the admin bar showing on login.wordpress.org, but regardless, In https://github.com/WordPress/wordpress.org/commit/99da786734fe0b0c9c82dee870f54f6bbda4c1b4 I've switched the theme to use the filter, which should 100% work to hide it I believe.

Reply:
I don't see it on the login page anymore 👍🏻